tests/test_skill_registry.py

```
@@ -0,0 +1,286 @@
+        assert "test-first-executor" in skills
+
+    def test_stripe_skills_relevant(self):
+        skills = TECH_SKILL_MAP.get("stripe", [])
+        assert "risky-module-review" in skills
+
+
+class TestWorkflowSkillMap:
+    """Tests for WORKFLOW_SKILL_MAP."""
+
+    def test_all_values_are_lists(self):
+        for wf, skills in WORKFLOW_SKILL_MAP.items():
```

Ruff (0.15.15)

[warning] Loop control variable `wf` not used within loop body

Rename unused `wf` to `_wf`

(B007)